README.md

```diff
@@ -1 +1,198 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
You did a really good job using the Microverse README template 😄 and it will be really nice for you to remove these commented lines of code as well as other commented codes in your README file to make code clean and clear to everybody 👍